src/app/actions/board.actions.ts

```diff
@@ -0,0 +1,45 @@
+import { Action } from '@ngrx/store';
+import { BoardState } from '../states/board.state';
+
+export const GET = '[board] Get';
+export const GET_SUCCESS = '[board] GetSuccess';
+export const GET_ERROR = '[board] GetError';
+export const GET_BOARD_URL = '[board] GetBoardapiUrl';
```

Comment:
We don't need an action for this. The source of API URL is likely going to be changed from space template response to space response. We can make two consecutive service call from the effect to fetch board.